### DISCUSSION TITLE
FormRender Mobile 欢迎体验 beta版

### DISCUSSION THREAD

**Discussion by lhbxs** [alibaba/x-render#1173] [General]
[使用文档地址](https://xrender.fun/form-render-mobile)

FormRender Mobile beta版本发布，欢迎大家来尝鲜！！！

相关需求可以在这个讨论组下提

**Comment by sukbearai** [comment]
兼容taro!!!

**Reply by sukbearai** [reply]
https://www.npmjs.com/package/@suwujs/vantui-form-render 适配了vantui，支持小程序和h5

**Comment by ghRocs** [comment]
只读以及屏蔽范围希望可以支持作用在表单的字段域级别，而非必须控制整个表单。

**Reply by lhbxs** [reply]
本身就支持控件级别的只读 配置 readOnly
